src/attributesDecorator.js

```diff
@@ -22,7 +23,11 @@ function attributesDecorator(declaredSchema, ErroneousPassedClass) {
           passedAttributes = {};
         }
 
-        instance.attributes = passedAttributes;
+        instance.attributes = Object.assign(
+          {},
```

Comment:
We won't need this object since `retrieveDefaultValues()` already creates an empty one.